src/components/views/elements/EditHistoryMessage.js

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Would the `messages` directory be a better fit? (The component directories are all a bit haphazard...)